Source/WebKit/CMakeLists.txt

```diff
@@ -588,6 +588,8 @@ set(WebKit_SERIALIZATION_IN_FILES
     WebProcess/UserContent/InjectUserScriptImmediately.serialization.in
 
     WebProcess/WebCoreSupport/WebSpeechSynthesisVoice.serialization.in
+
+    WebProcess/WebPage/wc/WCUpdateInfo.serialization.in
```

Comment:
Whoops caught this. It should just be in `PlatformWin.cmake`. It is not used by other ports

Reply:
Actually just delete this since you're appending to the `serialization.in` file.